content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -107,7 +107,8 @@ and the domain name for your cluster is `cluster.local`, then the Pod has a DNS
 
 `172-17-0-3.default.pod.cluster.local`.
 
-Any Pods exposed by a Service have the following DNS resolution available:
+If you're using [CoreDNS], Pods exposed by a Service will also have the following
+DNS resolution available:
 
 `pod-ipv4-address.service-name.my-namespace.svc.cluster-domain.example`.
```

Comment:
Extra credit: make it clear which of these are placeholders.
For example, this may work:
```suggestion
<tt><em>pod-ipv4-address</em>.<em>service-name</em>.<em>my-namespace</em>.svc.<em>cluster-domain.example</em>.</tt>
```

Reply:
I've included this change. But now this one line is inconsistent with the other DNS record examples in this file. Should I make another PR to do this for all of them?

Reply:
<img width="640" alt="image" src="https://github.com/user-attachments/assets/c7797e52-e258-4dbe-9301-b29f9582107f" />

hm not sure it did what we want?

Reply:
Ah `<em>` italicizes. This is how Chrome shows `<strong>`. Is this better visually?

<img width="479" alt="image" src="https://github.com/user-attachments/assets/58dc3071-ac50-4a83-be55-5959f28ed452" />

Reply:
I'm happy that this looked how I intended.